vscode/src/webview/docview.tsx

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Markdown } from "qsharp-lang/ux";
+
+export function DocumentationView(props: { contentToRender: string }) {
```

Comment:
If this component isn't going to add any functionality or formatting, just use the `Markdown` component directly. This component doesn't add anything.

Reply:
This is the component that will be used to filter content later. I placed content concatenation here as this is the only processing that we have at the moment.